forge/forge/tvm_to_python.py

```diff
@@ -1239,6 +1239,8 @@ def populate_cast_args(graph, nid, compiler_cfg):
     node = graph["nodes"][nid]
     args = []
     dtype = node["attrs"]["dtype"][0][0]
+    if dtype == "uint1":
```

Comment:
Why can't we use uint1, and have to use bool precisely? Those are the same, but why true/false vs 1/0? 

Can we do this in TVM? AFAIK there is the logic that converts dtypes from pytorch to Forge dtypes (and vice verse). It seems like that would be the proper place to do so.

Reply:
@nvukobratTT ,We can't use uint1 as it was throwing `AttributeError: module 'torch' has no attribute 'uint1'` and currently there is support only for the dtypes mentioned [here](https://github.com/tenstorrent/tt-tvm/blob/e91c07cfe4710c72a9976c791205032aff60d52b/python/tvm/relay/frontend/pytorch.py#L2451).

Reply:
@nvukobratTT , it has been changed that way because uint1 is not supported and it was throwing error `AttributeError: module 'torch' has no attribute 'uint1'`

I will check it out and let you know about the tvm part.